cnv/cnv_node_network/cnv-updating-node-network.adoc

```diff
@@ -0,0 +1,16 @@
+[id="cnv-updating-node-network"]
+= Updating node network configuration
```

Comment:
Since every article on the docs acts as standalone, would it make sense to have the nmstate introducion here as well? Or is there an expectation that users will read those articles in order?

Reply:
Have added the about-nmstate module to this assembly as well now